src/qt/locale/bitcoin_pt_BR.ts

```diff
@@ -680,11 +680,11 @@
     </message>
     <message>
         <source>This label turns red if any recipient receives an amount smaller than the current dust threshold.</source>
-        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que que o dust.</translation>
+        <translation>Este texto fica vermelho se qualquer destinatário receber uma quantidade menor que a poeira.</translation>
```

Comment:
For Brazilian Portuguese, I believe that an easy interpretation would be a percentage and not dust!